### PR TITLE
feat: Xamarin Wizard

### PR DESCRIPTION
**Pull request by lucas-zimerman**
The following pull request adds to the doc a Wizard for Xamarin Forms.

### CODE REVIEW

File: src/wizard/dotnet/xamarin.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+# Or using .NET Core CLI
+dotnet add package Sentry.Xamarin.Forms -v 1.0.0-alpha.2
+```
+
+<div class="alert alert-info" role="alert"><h5 class="no_toc">Using .NET Framework prior to 4.6.1?</h5>
+    <div class="alert-body content-flush-bottom">
+        <a href="https://docs.sentry.io/clients/csharp/">Our legacy SDK</a> supports .NET Framework as early as 3.5.
+    </div>
+</div>
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:22]: Don't think this note is relevant

```suggestion
```

File: src/wizard/dotnet/xamarin.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+{
+    protected override void OnCreate(Bundle savedInstanceState)
+    {
+        SentrySdk.Init(o =>
+        {
+			// Add the following line:
+            o.Dsn = "___PUBLIC_DSN___";
+            o.CacheDirectoryPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+            o.AddIntegration(new SentryXamarinFormsIntegration());
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:42]: Now that I see this, a proposed new API could be done with extension methods like:

```suggestion
            o.AddXamarinFormsIntegration();
```

File: src/wizard/dotnet/xamarin.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        });
+        ...
+```
+
+### iOS
+Initialize the SDK on your AppDelegate.cs
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:48]:
```suggestion
Initialize the SDK on your `AppDelegate.cs`
```

File: src/wizard/dotnet/xamarin.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+```csharp
+public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsApplicationDelegate
+{
+    public override bool FinishedLaunching(UIApplication app, NSDictionary options)
+    {
+        SentrySdk.Init(o =>
+        {
+			// Add the following line:
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:57]:
```suggestion
```

File: src/wizard/dotnet/xamarin.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            o.Dsn = "___PUBLIC_DSN___";
+            o.CacheDirectoryPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+            o.AddIntegration(new SentryXamarinFormsIntegration());
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:60]: You could run this: `o.CacheDirectoryPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);` inside your integration the extension method I suggested above and avoid requiring this configuration.

```suggestion
            o.CacheDirectoryPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
            o.AddIntegration(new SentryXamarinFormsIntegration());
```

File: src/wizard/dotnet/xamarin.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        });
+        ...
+```
+
+### UWP
+Initialize the SDK on App.xaml.cs.
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:66]:
```suggestion
Initialize the SDK on `App.xaml.cs`.
```

File: src/wizard/dotnet/xamarin.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    sealed partial class App : Application
+    {
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+        SentrySdk.Init(o =>
+        {
+			// Add the following line:
+            o.Dsn = "___PUBLIC_DSN___";
+            o.AddIntegration(new SentryXamarinFormsIntegration());
+        });
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:80]: Tabs and spaces mixed
missing identatation

```suggestion
            SentrySdk.Init(o =>
            {
                o.Dsn = "___PUBLIC_DSN___";
                o.AddIntegration(new SentryXamarinFormsIntegration());
            });
```

File: src/wizard/dotnet/xamarin.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+The package requires the following versions or newer:
+
+* Tizen 4.0 (for Tizen)
+* Xamarin.Android 9.0 (for Android)
+* Xamarin.iOS 10.14 (for iOS)
+* Universal Windows Platform 10.0.16299 (for UWP)
+* Xamarin.Forms 4.6.0.726
+* Xamarin.Essentials 1.4.0
+* Sentry 3.0.0
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:98]: If these are dependencies of the package, why list it here? 
It won't compile if they depend on a lower version.

File: src/wizard/dotnet/xamarin.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+
+## Limitations
+
+There are no line numbers on stack traces for UWP and in release builds for Android and iOS, furthermore, mono symbolication is not yet supported.
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:102]:
```suggestion
## Limitations

There are no line numbers on stack traces for UWP and in release builds for Android and iOS.
```

File: src/wizard/dotnet/xamarin.md (hunk 5 above)
Comment (lucas-zimerman) [reply src/wizard/dotnet/xamarin.md:60]: so the user can't opt out of using Cache?

File: src/wizard/dotnet/xamarin.md (hunk 1 above)
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:22]: This isn't needed on xamarin

File: src/wizard/dotnet/xamarin.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+### Android
+Initialize the SDK on your MainActivity.
+
+```csharp
+public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+{
+    protected override void OnCreate(Bundle savedInstanceState)
+    {
+        SentrySdk.Init(o =>
+        {
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:38]: Changing now

File: src/wizard/dotnet/xamarin.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+---
+
+## Install the NuGet package
+
+```shell
+# Using Package Manager
+Install-Package Sentry.Xamarin.Forms -Version 1.0.0-alpha.3
+
+# Or using .NET Core CLI
+dotnet add package Sentry.Xamarin.Forms -v 1.0.0-alpha.3
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:15]: This works with Xamarin?

**Review by bruno-garcia — APPROVED** [review]
LGTM! 1 nit left

File: src/wizard/dotnet/xamarin.md (hunk 11 above)
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:15]:
```suggestion
```

File: src/wizard/dotnet/xamarin.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+Initialize the SDK on your `MainActivity`.
+
+```csharp
+public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+{
+    protected override void OnCreate(Bundle savedInstanceState)
+    {
+		SentryXamarin.Init(options =>
+		{
+			options.Dsn = "__YOUR__DSN__";
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:30]:
```suggestion
			options.Dsn = "___PUBLIC_DSN___";
```

File: src/wizard/dotnet/xamarin.md [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+```csharp
+    sealed partial class App : Application
+    {
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
+        {
+			SentryXamarin.Init(options =>
+			{
+				options.Dsn = "__YOUR__DSN__";
+			});
+			...
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:64]:
```suggestion
            SentryXamarin.Init(options =>
            {
                options.Dsn = "___PUBLIC_DSN___";
            });
```

File: src/wizard/dotnet/xamarin.md [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+```csharp
+public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsApplicationDelegate
+{
+    public override bool FinishedLaunching(UIApplication app, NSDictionary options)
+    {
+		SentryXamarin.Init(options =>
+		{
+			options.Dsn = "__YOUR__DSN__";
+		});
+        ...
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:47]:
```suggestion
        SentryXamarin.Init(options =>
        {
            options.Dsn = "___PUBLIC_DSN___";
        });
```

File: src/wizard/dotnet/xamarin.md (hunk 13 above)
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:64]:
```suggestion
            SentryXamarin.Init(options =>
            {
                options.Dsn = "___PUBLIC_DSN___";
            });
```

**Review by bruno-garcia — APPROVED** [review]
Removed tabs in favor of spaces. Added `___PUBLIC_DSN___` note the 3 `_` which gets replaced by Sentry with your DSN

File: src/wizard/dotnet/xamarin.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+```csharp
+public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+{
+    protected override void OnCreate(Bundle savedInstanceState)
+    {
+		SentryXamarin.Init(options =>
+		{
+			options.Dsn = "___PUBLIC_DSN___";
+		});
+        ...
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:32]:
```suggestion
        SentryXamarin.Init(options =>
        {
            options.Dsn = "___PUBLIC_DSN___";
        });
```

File: src/wizard/dotnet/xamarin.md [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+doc_link: https://docs.sentry.io/platforms/dotnet/guides/xamarin/
+support_level: alpha
+type: library
+---
+
+## Install the NuGet package
+
+```shell
+# Using Package Manager
+Install-Package Sentry.Xamarin.Forms -Version 1.0.0-alpha.3
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:12]:
```suggestion
# For Xamarin.Forms
Install-Package Sentry.Xamarin.Forms -Version 1.0.0-alpha.3

# If you are not using Xamarin.Forms, but only Xamarin:
Install-Package Sentry.Xamarin -Version 1.0.0-alpha.3
```

And then also explain below what needs to be done if adding Xam.Forms (the integration thing)

File: src/wizard/dotnet/xamarin.md [hunk 17]
```diff
@@ -9,12 +9,17 @@ type: library
 
 ```shell
 # Using Package Manager
```
Comment (bruno-garcia) [inline src/wizard/dotnet/xamarin.md:11]:
```suggestion
```

**Review by bruno-garcia — APPROVED** [review]

**Pull request merged**